source/core/ut_utils.pks

```diff
@@ -293,5 +293,11 @@ create or replace package ut_utils authid definer is
   */
   function to_xml_number_format(a_value number) return varchar2;
 
+  /*It takes a collection of type ut_varchar2_list and it trims the characters passed as arguments for every element*/
+  function trim_list_elements(a_list IN ut_varchar2_list, a_characters_to_trim in varchar2) return ut_varchar2_list;
```

Comment:
I would rename the parameter to be named `a_regexp_to_trim`, give it a default value of `[:space:]` or even consider removing the parameter as we don't have a use-case for trimming anything else than any blank characters at the moment.

Reply:
It's good if I add the default and change the parameter name. But I thinks I can be useful if you want to trim -?(),.;  something like that for a collection someday